client/post-editor/editor-categories-tags/accordion.jsx

```diff
@@ -163,27 +141,32 @@ module.exports = React.createClass( {
 			subtitlePieces.push( tagsSubtitle );
 		}
 
-		if ( ! isPage( this.props.post ) && this.isCategoriesLoading() ) {
-			return this.translate( 'Loading…' );
+		if ( isPost && this.isLoading() ) {
+			return translate( 'Loading…' );
```

Comment:
I've suggested to eliminate the "Loading" label in the Sharing accordion in #7164 since we're not really using it consistently (and I don't think it adds much). Do you think we should do the same here?